docs/template_extension/extension.cpp

```diff
@@ -111,17 +111,3 @@ InferenceEngine::ILayerImpl::Ptr Extension::getImplementation(const std::shared_
 // Generate exported function
 IE_DEFINE_EXTENSION_CREATE_FUNCTION(Extension)
 //! [extension:CreateExtension]
-
-INFERENCE_EXTENSION_API(InferenceEngine::StatusCode)
-InferenceEngine::CreateExtension(InferenceEngine::IExtension*& ext, InferenceEngine::ResponseDesc* resp) noexcept {
-    try {
-        ext = new Extension();
-        return OK;
-    } catch (std::exception& ex) {
-        if (resp) {
-            std::string err = ((std::string) "Couldn't create extension: ") + ex.what();
-            err.copy(resp->msg, 255);
-        }
-        return InferenceEngine::GENERAL_ERROR;
-    }
-}
```

Comment:
as far as I remember, it's used in tests to check that old path still works

Reply:
I would like to remove it (and return back in the future). Because I don't know how new way works (looks like we have a typo in the master branch) and I didn't find usages of new API..

Reply:
Reverted